ledger-service/http-json-cli/src/main/scala/com/daml/http/OptionParser.scala

```diff
@@ -47,6 +49,11 @@ class OptionParser(getEnvVar: String => Option[String], supportedJdbcDriverNames
 
   override def renderingMode: RenderingMode = RenderingMode.OneColumn
 
+  override def displayToOut(msg: String): Unit = logger.info(msg)
+  override def displayToErr(msg: String): Unit = logger.error(msg)
+  override def reportError(msg: String): Unit = logger.error(msg)
+  override def reportWarning(msg: String): Unit = logger.warn(msg)
```

Comment:
Not sure CLI output needs to go via the logger. Structured logging is really mainly useful for long-running stuff not when you messed up CLI flags and the thing didn’t even start.

Reply:
Will remove this too then

Reply:
And this is what I talked about yesterday with you in Slack :D